lib/resources/majors.tsx

```diff
@@ -868,7 +881,6 @@ const bsBME: Major = {
       description:
         'Students are required to take at least one semester of programming from a select set of gateway computing courses.',
       criteria: 'EN.500.112[C]^OR^EN.500.113[C]^OR^EN.500.114[C]',
-      double_count: ['N/A'],
```

Comment:
Instead of removing this, can we make it an empty array?

Reply:
we could, but I prefer having !double_count as no double_count. Less typing :/

Reply:
Resolved offline.